Create string formatter

aaossa opened the pull request:
Creates a new class with a static method to convert strings (in this case, names) into valid paths.

aaossa committed:
Create format.js

This file should contain the static methods used to format strings. In
this commit, it includes a method to convert strings into valid paths

Signed-off-by: Antonio Ossa <aaossa@uc.cl>

aaossa committed:
Use format module to convert names to paths

In each file two lines are changed:
1. Require format module
2. Convert object name to a valid path

Signed-off-by: Antonio Ossa <aaossa@uc.cl>

negebauer reviewed (changes requested):
Let's also modify this
https://github.com/open-source-uc/sincding/blob/d581b3c314d38598eeea6f24f08d5466fbf24612/course.js#L15

aaossa committed:
Set course name directly

Now we don't need to replace the characters, because the new format
methods will handle that

Signed-off-by: Antonio Ossa <aaossa@uc.cl>
